README.md

```diff
@@ -11,15 +11,16 @@
 
 # Table of contents
 1. [Project Specs](#coinspecs)
-2. [Downloads](#downloads)
-3. [Useful Links](#usefullinks)
-4. [How to Compile Qwertycoin](#howtocompile)
+2. [How to Compile Qwertycoin](#howtocompile)
     1. [Qwertycoin for Linux](#build-linux)
     2. [Qwertycoin for Windows](#build-windows)
-    3. [Qwertycoin for Apple](#build-apple)
+    3. [Qwertycoin for MacOS](#build-apple)
```

Comment:
```suggestion
    3. [Qwertycoin for macOS](#build-apple)
```

Reply:
Apple officially calls it macOS (starting with lower case **m**). I think we should follow this pattern 🙂